docs/guides/cache/index.md

```diff
@@ -129,47 +127,27 @@ with both parameters to find the results that work best for you.
 
 ### Cache compression
 
-Since `registry` cache image is a separate export artifact from the main build
-result, you can specify separate compression parameters for it. These parameters
-are similar to the options provided by the `image` exporter. While the default
-values provide a good out-of-the-box experience, you may wish to tweak the
-parameters to optimize for storage vs compute costs.
+<!-- FIXME: this link won't work on docs.docker.com -->
+
+The cache compression options are the same as the
+[exporter compression options](../exporters/index.md#compression).
```

Comment:
Let's use absolute link for now and merge this PR; I will create a follow-up PR for restructuring the files to match the downstream.

```suggestion
[exporter compression options](https://docs.docker.com/build/building/exporters/).
```